apps/frontend/src/app/5_pages/ZeroPage/ZeroPage.tsx

```diff
@@ -125,130 +128,108 @@ export const ZeroPage: FC = () => {
-                        borrowingRate={decimalic(
-                          fees?.borrowingRate()?.toString(),
-                        )}
-                      />
-                    )}
-                    {open && hasLoc && (
-                      <AdjustCreditLine
-                        existingCollateral={collateral}
-                        existingDebt={debt}
-                        rbtcPrice={price}
-                        borrowingRate={decimalic(
-                          fees?.borrowingRate()?.toString(),
-                        )}
-                        onSubmit={handleTroveSubmit}
-                      />
-                    )}
-                  </DialogBody>
-                </Dialog>
-
-                <GettingStartedPopup
-                  isOpen={openStartedPopup && !isTxOpen}
-                  onConfirm={toggleStartedPopup}
-                />
-
-                <Dialog
-                  width={DialogSize.md}
-                  isOpen={openClosePopup}
-                  disableFocusTrap
-                >
-                  <DialogHeader
-                    title={t(translations.zeroPage.loc.close)}
-                    onClose={toggleClosePopup}
-                  />
-                  <DialogBody>
-                    <CloseCreditLine
-                      onSubmit={handleTroveClose}
-                      creditValue={debt.sub(LIQUIDATION_RESERVE_AMOUNT)}
-                      collateralValue={collateral}
-                    />
-                  </DialogBody>
-                </Dialog>
-              </>
+            onClose={toggle}
+          />
+          <DialogBody>
+            {open && !hasLoc && (
+              <OpenCreditLine
+                onSubmit={handleTroveSubmit}
+                rbtcPrice={price}
+                borrowingRate={borrowingRate}
+              />
+            )}
+            {open && hasLoc && (
+              <AdjustCreditLine
+                existingCollateral={collateral}
+                existingDebt={debt}
+                rbtcPrice={price}
+                borrowingRate={borrowingRate}
+                onSubmit={handleTroveSubmit}
+              />
             )}
-          </Await>
-        </React.Suspense>
+          </DialogBody>
+        </Dialog>
+
+        <GettingStartedPopup
+          isOpen={openStartedPopup && !isTxOpen}
+          onConfirm={toggleStartedPopup}
+        />
+
+        <Dialog width={DialogSize.md} isOpen={openClosePopup} disableFocusTrap>
+          <DialogHeader
+            title={t(translations.zeroPage.loc.close)}
+            onClose={toggleClosePopup}
+          />
+          <DialogBody>
+            <CloseCreditLine
+              onSubmit={handleTroveClose}
+              creditValue={debt.sub(LIQUIDATION_RESERVE_AMOUNT)}
+              collateralValue={collateral}
+            />
+          </DialogBody>
+        </Dialog>
       </div>
     </>
   );
 };
+
+export default withDeferedLoaderData(ZeroPage, 'deferedData');
```

Comment:
```suggestion
export default withDeferredLoaderData(ZeroPage, 'deferredData');
```

See comment in `withDeferedRouterData.tsx`